GetStream/stream-chat-android
# [3208] Improve sending messages with file attachments

### 🎯 Goal

Ensure that ephemeral messages are replaced with regular ones without a delay.

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/158485771-7479c3be-fd61-4df9-8c2b-34fef8c94a2a.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/158485744-1f98f69a-2613-4393-813f-b248a28215f9.mp4" controls="controls" muted="muted" />
</td>
</tr>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/158485756-1ba5b683-f565-4fec-b55d-1e2947fa3c01.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/158485718-bf42ab2a-6eed-42b1-b1ae-1ca0003f8e62.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Send a message with file or image attachments
2. Observe that there is no flicker

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
